tests/virtual_machines_test.go

```diff
@@ -82,49 +82,24 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				return len(vmList.Items)
 
 			}, timeout, pollingInterval).Should(Equal(0), "failed to remove all vm objects")
-
-			// remove all the labels from the test namespaces
-			for _, namespace := range []string{TestNamespace, OtherTestNamespace} {
-				err = cleanNamespaceLabels(namespace)
-				Expect(err).ToNot(HaveOccurred(), "should be able to remove the namespace labels")
-			}
-		})
-
-		Context("When the client wants to create a vm on an opted-out namespace", func() {
-			It("should create a vm object without a MAC assigned when no label is set to namespace", func() {
-				//remove namespace opt-in labels
-				By("removing the namespace opt-in label")
-				err := cleanNamespaceLabels(TestNamespace)
-				Expect(err).ToNot(HaveOccurred(), "should be able to remove the namespace labels")
-
-				vm := CreateVmObject(TestNamespace, false, []kubevirtv1.Interface{newInterface("br", "")},
-					[]kubevirtv1.Network{newNetwork("br")})
-
-				err = testClient.VirtClient.Create(context.TODO(), vm)
-				Expect(err).ToNot(HaveOccurred())
-				Expect(vm.Spec.Template.Spec.Domain.Devices.Interfaces[0].MacAddress).Should(Equal(""))
-			})
 		})
 
 		Context("When kubemacpool is disabled on a namespace", func() {
```

Comment:
It is enabled though, isn't it?

Reply:
it's not, notice we are opting out the namespace below

Reply:
Aha, I did not check the It. Context should be defined by its BeforeEach. Otherwise it is just a text with no guarantees that the actual environment matches it. Would you move the setup to BeforeEach?

Reply:
Done